Dockerfile

```diff
@@ -18,6 +18,7 @@ COPY . .
 
 # Build
 ARG OPERATOR_VERSION
+ARG TEMPO_VERSION
```

Comment:
@rubenvp8510 do you know how `OPERATOR_VERSION` is passed in GHA when the image is published? We might need to update the workflow to include the tempo version